lib/datasource/metadata.ts

```diff
@@ -189,22 +189,22 @@ function massageTimestamps(dep: ReleaseResult): void {
 }
 
 export function addMetaData(
-  dep?: ReleaseResult,
-  datasource?: string,
-  lookupName?: string
+  dep: ReleaseResult,
+  datasource: string,
+  lookupName: string
 ): void {
-  if (!dep) {
```

Comment:
Unlikely to happen since the only call returns early for nullable `dep` and `datasource` values